.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Ask a question
+    url: https://github.com/vmware-labs/ml-conversational-analytic-tool/discussions
```

Comment:
Thank you @ericwb 
LGTM, except that this link returns 404. I am  not sure if this is ok at that stage, but wanted no mention it 

I did a research and I found that a GitHub feature - `GitHub discussions` could be easily enabled for the project 
https://docs.github.com/en/discussions

Reply:
Hmm, weird, the link works for me: https://github.com/vmware-labs/ml-conversational-analytic-tool/discussions. Can you try again?

Reply:
the link also works for me